juju/juju
# [JUJU-1453] Test charm cleanup

Continued work that is stable for cleaning up charms in our testing infra.

- Upgrading charms for charmcraft/charmhub support.
- Adding LTS ubuntu releases including jammy support.
- Adding support for other arch's besides amd64
- Moving tests to use cloud charms from Charmhub and not performing local deploys unless that is the test's purpose.

## Checklist


## QA steps

- Verify uploaded charms in Charmhub
- Run affected tests suits locally for successful test run.

## Documentation changes

N/A

## Bug reference

N/A
